components/src/histogram.tsx

```diff
@@ -19,18 +19,20 @@
  *
  */
 
+import { getBooleanAttributes } from './utils';
+
 function IobioHistogram({
 	brokerKey,
 	color,
-	ignoreOutliers,
+	ignoreOutliers = false,
 	title,
 }: {
 	brokerKey: string;
 	color?: string;
 	ignoreOutliers?: boolean;
 	title?: string;
 }) {
-	const outlierProp = ignoreOutliers ? { 'ignore-outliers': true } : {};
+	const outlierProp = getBooleanAttributes({ ignoreOutliers });
```

Comment:
this is no longer "outlierProp" , it's the html boolean attributes for this component eg. <iobio... {...booleanAttributes}/>

Reply:
Updated